src/NetworkConfig.ts

```diff
@@ -107,8 +107,6 @@ export const defaultMainnetNetworkConfig: NetworkConfig = {
         generationHash: 'ED5761EA890A096C50D3F50B7C2F0CCB4B84AFC9EA870F381E84DDE36D04EF16',
     },
     nodes: [
-        { friendlyName: 'api-01-dhealth', roles: 2, url: 'http://api-01.dhealth.cloud:3000' },
-        { friendlyName: 'api-02-dhealth', roles: 2, url: 'http://api-02.dhealth.cloud:3000' },
         { friendlyName: 'dual-01-dhealth', roles: 2, url: 'http://dual-01.dhealth.cloud:3000' },
```

Comment:
LGTM, but does it make sense to put these run-time constants into the config db?

Reply:
> LGTM, but does it make sense to put these run-time constants into the config db?

Since this is a public repo, I think including a config set from the config db may be inconvenient. As either when user build the project they would need to have the Firebase account associated with our Firebase project, and it'll also be not possible for users to read the code for dHealth network config.

Reply:
Or I think we need to somehow update our central config database to allow:
- Public access records - which allows no authentication, everyone can access it e.g. via urls.
- Separation between different projects and users.